src/backend/main.ts

```diff
@@ -980,7 +980,7 @@ addHandler(
     })
 
     const abortMessage = () => {
-      notify({ title, body: i18next.t('notify.install.canceled') })
+      notify({ title, body: i18next.t('notify.importing.failed') })
```

Comment:
```suggestion
      notify({ title, body: i18next.t('notify.import.failed', 'Import failed') })
```

the other keys under `notify` don't end up with `ing`, and it's a good idea to add the default value when adding a new key